tahoe_sites/api.py

```diff
@@ -317,3 +317,56 @@ def get_organization_by_course(course_id):
             active=True,
         ).values_list('organization_id', flat=True)
     ).get()
+
+
+def get_organization_user_by_email(email, organization, fail_if_inactive=False):
+    """
+    Get the user owning the given email address in the given organization. With an option to return None if
+    the user is inactive
+
+    :param email: user's email to search for
+    :param organization: organization to filter on
+    :param fail_if_inactive: when <True>; the method will fail if the user is inactive. (default is False)
+    :return: user object
+    """
+    if fail_if_inactive:
+        extra_params = {'user__is_active': True}
+    else:
+        extra_params = {}
+
+    return get_user_model().objects.get(
+        pk__in=UserOrganizationMapping.objects.filter(organization=organization, **extra_params).values('user_id'),
+        email=email,
+    )
+
+
+def is_exist_organization_user_by_email(email, organization, must_be_active=False):
+    """
+    Check if the user owning the given email address in the given organization exists or Not
+
+    :param email: user's email to search for
+    :param organization: organization to filter on
+    :param must_be_active: when <True>; the method will return (False) if the user is inactive. (default is False)
+    :return: boolean
+    """
+    try:
+        user = get_organization_user_by_email(email=email, organization=organization, fail_if_inactive=must_be_active)
+    except get_user_model().DoesNotExist:
+        user = None
+    return user is not None
+
+
+def get_admin_users_queryset_by_email(email):
```

Comment:
mark it clearly 
make it obvious from the function name that it's a temporary function.

```suggestion
def deprecated_get_admin_users_queryset_by_email(email):
```

Reply:
done

Reply:
actually, the change was lost by mistake. fixed [here](https://github.com/appsembler/tahoe-sites/pull/49)